easy-ui-react/src/Notification/Notification.module.scss

```diff
@@ -0,0 +1,83 @@
+@use "../styles/common" as *;
+
+// prettier-ignore
+.statusPromotional {
+  @include component-token("notification", "color", theme-token("color.text.inverse"));
+  @include component-token("notification", "color.background", theme-token("color.background.primary"));
+}
+
+// prettier-ignore
+.statusSuccess {
+  @include component-token("notification", "color", theme-token("color.text.success.bold"));
+  @include component-token("notification", "color.background", theme-token("color.background.success.subtle"));
+}
+
+// prettier-ignore
+.statusNeutral {
+  @include component-token("notification", "color", theme-token("color.text.secondary.bold"));
+  @include component-token("notification", "color.background",theme-token("color.background.secondary.subtle"));
+}
+
+// prettier-ignore
+.statusWarning {
+  @include component-token("notification", "color", theme-token("color.text.inverse.inverse.bold"));
+  @include component-token("notification", "color.background", theme-token("color.background.warning.subtle"));
+}
+
+// prettier-ignore
+.statusError {
+  @include component-token("notification", "color", theme-token("color.text.danger.bold"));
+  @include component-token("notification", "color.background", theme-token("color.background.danger.subtle"));
+}
+
+.typeToast {
+  display: inline-flex;
+}
+
+.typeAlert {
+  display: flex;
+  width: 100%;
+}
+
+// prettier-ignore
+.Notification {
+  @include component-token("notification", "border_radius", design-token("shape.border_radius.base"));
+  @include component-token("notification", "max-width", 1200px);
+  align-items: center;
+  justify-content: space-between;
+  margin: auto;
+  padding: design-token("space.1");
+  border-radius: 0 0 component-token("notification", "border_radius") component-token("notification", "border_radius");
+  max-width: component-token("notification", "max-width");
+  color: component-token("notification", "color");
+  background-color: component-token("notification", "color.background");
+}
+
+.iconText {
+  display: inline-flex;
+  align-items: center;
+  gap: design-token("space.1");
+}
+
+// prettier-ignore
+.closeButton {
+  border: 0;
+  padding: design-token("space.0");
+  margin: design-token("space.0.5") design-token("space.1") design-token("space.0.5") design-token("space.2");
+  background-color: component-token("notification", "color.background");
+  color: component-token("notification", "color.text");
+  cursor: pointer;
+}
+
+.notificationRegion {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.notificationContainer {
+  position: fixed;
+  z-index: 999999;
+  width: 100%;
+  top: 0;
```

Comment:
**issue:** without a `left: 0;` property, the notification will be left-aligned to whatever the body padding is, which may not be what we want.

i notice in Storybook that because there is no `left: 0` it exceeds past the story window and cuts off the right-most edge:

<img width="1078" alt="image" src="https://github.com/EasyPost/easy-ui/assets/752942/dc111595-904f-4ade-a651-b3617fd73234">

Reply:
ah interesting, didn't know that, added it!